src/main/frontend/src/css/App.css

```diff
@@ -0,0 +1,5 @@
+a.tab:link,
+a.tab:visited {
+    color: #007bff;
```

Comment:
Hack to unset global `a:link` and `a:visited`.

Reply:
Yeah, that's a known PITA, I want to fix that